home/templates/home/explore.html

```diff
@@ -2,14 +2,20 @@
 {% load static %}
 {% block content %}
 
-<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
-<script type="text/javascript" src="{% static 'home/js/explore_page_js.js' %}"></script>
 <link rel="stylesheet" type="text/css" href="{% static 'home/CSS/explore-page.css' %}">
 <body class="main-bg">
 
 <form method="GET" class="container">
 <div class="m-5" >
 
+    {% if 'tag' in request.GET %}
```

Comment:
Its better to leave request parsing to the view function and make the template only rely on the context. At the very least it makes it easier to test the logic without having to do fragile string matching on the response HTML.